jest.config.js

```diff
@@ -0,0 +1,43 @@
+let mappedModule;
+switch (process.env.TEST_ENV) {
+  case "PRODUCTION":
+    mappedModule = "<rootDir>/dist/cloudinary-react.js";
+    break;
+  default:
+    mappedModule = "<rootDir>/src/index.js";
```

Comment:
Does this mean you're preventing any import from ./src/some/path and you're only allowing imports from ./src/index.js?

If so, this might be problematic - there should be plenty of things you'll want to test in a unit-test environment that aren't exported through index.js, this rule would prevent that.

If we want to run our tests on the production code, we should maybe add a dedicated test for ./dist?

Reply:
mappedModule replaces 'cloudinary-react' to be './src/index.js' or './dist/cloudinary-react.js'
depending on if TEST_ENV==="PRODUCTION", which it is when testing dist.
this way test:unit imports from src, while test:build imports from dist